sbin/setkey/setkey.8

```diff
@@ -281,7 +284,7 @@ and they cannot be used.
 .Pp
 .It Ar extensions
 take some of the following:
-.Bl -tag -width Fl natt_mtu -compact
+.Bl -tag -width Fl -compact
```

Comment:
Aha!  I've been puzzled about this for a while.  That `-width Fl` seems to be the same as `-width 12n`, but why?

I see in the docs
> A macro invocation, which selects a predefined
> width associated with that macro.  The most
> popular is the imaginary macro Ds, which
> resolves to 6n.

but is there a list of those predefined widths anywhere?

Reply:
Please let me know if you find it 😅

Reply:
Hmm.  If that list either doesn't exist or is hard to find, then should we be adding a `-width Fl` to the project?  What would you feel about `-width 12n`?  I think that's more clear about what's happening.

Reply:
It's good mdoc(7) which is not hard to find in the tree with `% git grep "width Fl"` or test it in the linter.

When I was first new to technical writing in the src tree I tried to use `(number)n` and was asked me not to because it's not obvious which one is the long one when the list gets big. And you have to count or make the computer count.

Reply:
I agree that there's lots of instances of `-width Fl`, but that's not a guarantee of quality.

There's (at least) four options here:
- `-width Fl`
- `-width 12n`
- `-width indent-two`
- `-width "-hwif ifname"` (or something similar)

If `12n` was discouraged because it's not obvious which argument is the longest one, then `Fl` surely suffers from the same problem.  Also, this particular list contains an option which is 33 chars long, and another which is 18 chars.  It's not clear to me why `Fl` is the best option in this case.

That said, `-width` lengths are fairly low on the list of things that I'm concerned with, so I'll bow out and leave it to others to discuss further.